cmd/snap/cmd_quota.go

```diff
@@ -118,13 +177,64 @@ func parseQuotas(maxMemory string) (*client.QuotaValues, error) {
 		mem = value
 	}
 
+	if cpuMax != "" {
+		countValue, percentageValue, err := parseCpuQuota(cpuMax)
+		if err != nil {
+			return nil, err
+		}
+		if percentageValue > 100 || percentageValue <= 0 {
+			return nil, fmt.Errorf("cpu quota percentage must be between 1 and 100")
+		}
+
+		cpuCount = countValue
+		cpuPercentage = percentageValue
+	}
+
+	if cpuSet != "" {
+		cgv, err := getCGroupVersion()
+		if err != nil {
+			return nil, err
+		}
+		if cgv < 2 {
+			return nil, fmt.Errorf("cannot use --cpu-set with cgroup version %d", cgv)
```

Comment:
same here, I would remove `--cpu-set` and just say "cannot specify CPU sets with..."